discord.sh

```diff
@@ -5,6 +5,7 @@ socat $SOCAT_ARGS \
     UNIX-CONNECT:$XDG_RUNTIME_DIR/discord-ipc-0 \
     &
 socat_pid=$!
+disable-breaking-updates.py
```

Comment:
```suggestion
./disable-breaking-updates.py
```

Reply:
What is the benefit of this? If this is to avoid clashing with something in PATH, I should do this for the set-dark-theme.py script, right?

Reply:
The current working directory isn't usually in `PATH` at all, so I'm surprised it works as-is...

Reply:
It works because we put the script in `/app/bin`, see https://github.com/flathub/com.discordapp.Discord/blob/master/com.discordapp.Discord.json#L108 (same thing is done with this new script)

Reply:
OK, that makes sense; ignore this comment, then.